src/Controls/tests/Core.UnitTests/ToolbarTests.cs

```diff
@@ -56,6 +56,39 @@ public async Task TitleAndTitleViewAreMutuallyExclusive()
 			Assert.Equal("Test Title", toolbar.Title);
 		}
 
+		[Fact]
+		public void ToolbarTitle_UsesTabbedPageChildTitle()
+		{
+			var window = new Window();
```

Comment:
If you use TestWindow here that should take care of the life cycle events

Reply:
Changed to use TestWindow.